src/components/Common/Sidebar/ZekSidebar.vue

```diff
@@ -39,14 +39,15 @@
         <li
           v-if="sec.title"
           class="link-container"
+          :class="sec.title.isActive ? 'active-link' : ''"
           :style="sec.title.isActive && activeColor ? { color: activeColor } : ''"
         >
           <a
             v-if="sec.links && sec.links.length"
             href="javascript:"
             :title="sec.title.tooltip || sec.title.label"
             class="link title"
-            @click="sec.title.isExpanded = !sec.title.isExpanded"
+            @click="expandedStatus(i)"
```

Comment:
I understand that the `$emit` is needed, but why disallow the toggling from happening on click by default?? Which could also have been used in other projects

Reply:
I tried it on many project and it does not work also, it needs to Collapse the whole sidebar to affect

Reply:
I think it was also working on other projects. Even if it does not work I think an internal fix would be better rather than emitting an event and handling it from outside the component.